ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -116,3 +116,8 @@ port={{ dashd_port }}
 {% if dashd_seednode is not none and not dashd_isseednode %}
 addnode={{ dashd_seednode }}:{{ dashd_port }}
 {% endif %}
+
+{% if dashd_walletnode is not none and coinjoin_wallet_count is defined %}
+coinjoinmultisession=1
+coinjoinsessions=16
+{% endif %}
```

Comment:
Will this work on testnet if it is under the `[devnet]` section? Either way let's move it up in the file so the logic is clear to someone reading the conf